src/constants.js

```diff
@@ -1,6 +1,9 @@
 const
     PREFIX = 'octotree'
-
+  , DOMAINS = {
```

Comment:
Comma warnings can be turned off with 'laxcomma'.<br>'DOMAINS' is defined but never used.